ui/client/datasets/Flows.js

```diff
@@ -92,6 +92,7 @@ const BasicRegistrationFlow = {
       jobs: [
         {
           id: 'elwood_processors.run_elwood',
+          send_context: true,
```

Comment:
I took a peek in the original changes where we started using elwood instead of mixmasta, but this line was still missing before then. I'm missing some context. @Sorrento110 it would be helpful for me to understand why this wasn't needed before (did "mixmasta" cache the context previously?)

Let me know! Thanks.

Reply:
I think it has more to do with the change to elwood processors where there is now an else clause at the bottom of the code chunk that appends features when it can't find a qualfies object in the features object in the annotations: https://github.com/jataware/dojo-stack/blob/815a335cd37d1f4d419e3d93112e3532687219b8/tasks/tasks/elwood_processors.py#L259

Reply:
Aaah, I missed that completely. Thanks!

Reply:
I take it this send_context is not needed, but harmless, then.